pipelines.py

```diff
@@ -103,6 +108,31 @@ def unet_weighted_padded(config, train_mode):
     return unet_weighted_padded
 
 
+def unet_validation(config):
+    unet_validation = unet_weighted(config, False)
+    first_step = unet_validation.get_step('mask_resize')
+    first_step.input_steps = []
+    first_step.input_data = ['input']
```

Comment:
@taraspiotr name 'input' could be missleading since we are using this for metadata all over pipelines.py. Something like `unet_predictions` used as `input_data` seems more descriptive